lib/chef/resource/chef_client_cron.rb

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright:: 2020, Chef Software Inc.
+# Copyright:: 2020-2020, Chef Software Inc.
```

Comment:
?

Reply:
my vim script, never figured out what the regexp bug is there.

we could just delete all the dates.  they don't matter.

Reply:
I'm fine with just nuking them